src/badge/library.cairo

```diff
@@ -278,20 +245,11 @@ namespace CarbonableBadgeMinter {
     // Externals
     //
 
+    // @notice Claim a badge of the given type.
+    // @param badge_type badge type.

```

Comment:
Cap char for description, also try to add an article in descriptions such as "The badge type" instead of "badge type"